lang/java/reef-io/src/main/java/org/apache/reef/io/network/impl/NetworkServiceParameters.java

```diff
@@ -58,4 +59,7 @@
   public static class NetworkServiceExceptionHandler implements Name<EventHandler<?>> {
   }
 
+  @NamedParameter(doc = "name client")
+  public static class NetworkServiceNameClient implements Name<NameClient> {
```

Comment:
Why would we need a `Name` for the `NameClient`? Isn't it a matter of binding an implementation?